src/main.rs

```diff
@@ -74,8 +76,19 @@ const CRATE_BLACKLIST: [&'static str; 10] = [
 ];
 
 const RUSTC_SHIM_ENV_VAR_NAME: &'static str = "RLS_RUSTC_SHIM";
+const VERSION: &'static str = concat!(env!("CARGO_PKG_VERSION"), "-", include_str!(concat!(env!("OUT_DIR"), "/commit-info.txt")));
+const BRIEF_DESCRIPTION: &'static str =
```

Comment:
I don't think pulling this text into a const improves the readability of the code. Could you leave it inline please.

Reply:
Will do!